### DISCUSSION TITLE
Scan_Type

### DISCUSSION THREAD

**Discussion by MSAfify** [danielver02/ALPS#168] [Q&A]
Hi all,
Regarding scan_type=2, please let me know the format of angle $\theta_{k,B}$ to $|k|$ =range $_\textrm{f}$ that should be in the input file.

Also, it is possible to change the value of k_parallel while keeping k_perp and theta constants using scan_type=2 for kappa distribution, right?

Many thanks in advance!
Mahmoud

**Comment by kgklein** [comment]
Mahmoud-
For the scan_type=2, the code will take the current value of theta fixed, and adjust both kperp and kpar while scanning through steps in |k|=sqrt[kperp^2+kpar^2] to a final value that the user defines via the swf variable.

As theta depends on the ratio between kpar and kperp, keeping kperp fixed and varying kpar will necessarily vary theta. You can vary kperp with kpar fixed with scan_type 3 and vary kpar with kperp fixed with scan_type 4.

The choices of the underlying distribution don't impact the wavevector scans, so all of this will work for bimaxwellian, kappas, Juettner, or arbitrary distributions.
Cheers
Kris

**Reply by MSAfify** [reply]
Thanks!
